api/pom.xml

```diff
@@ -90,7 +90,7 @@
         <non.final>false</non.final>
         <skip.release.tests>false</skip.release.tests>
         <spec.version>1.1</spec.version>
-        <jakarta.json.version>1.1.5</jakarta.json.version>
+        <jakarta.json.version>2.0-SNAPSHOT</jakarta.json.version>
```

Comment:
Is this publicly available anywhere? Or are you building JSON-P API from source?

Reply:
@aguibert take a look at oss sonatype snapshot repo: https://oss.sonatype.org/service/local/repositories/snapshots/content/jakarta/json/jakarta.json-api/2.0-SNAPSHOT/

Reply:
I don't like dependencies on snapshots. Let's release beta of JSONP API with Jakarta changes before merging this PR.

Reply:
@m0mus are you saying we should release jsonp-2-rc1 this week and rc2 next week where the only difference is going to be updated parent pom? Is there a need to rush that much instead of having some time to review this and just "block" merging before required jsonp becomes available? Downstream projects can either wait or build what is required themselves to not be blocked...

Reply:
@lukasj New parent pom is released. We should release JSONP2 RC1 this week. There is no need to release RC2 next week.

Reply:
I do not see it at https://repo1.maven.org/maven2/org/eclipse/ee4j/project/ yet. Search in staging repos gives no result as well. Do you mean it's available in jakarta.oss.sonatype.org repo? That is the one I haven't checked yet.

Reply:
@Tomas-Kraus please check it. Something went wrong with the parent pom release yesterday.

Reply:
According to this https://github.com/eclipse-ee4j/jms-api/pull/253#issuecomment-574036342 version number has to be changed to `2.0.0-SNAPSHOT`

Reply:
would you mind filing an issue(s) to the appropriate project?

Reply:
Nothing went wrong, I did release to staging. Script for central release was never implemented for parent pom project. I did it today.
```
[INFO]  * Connected to Nexus at https://jakarta.oss.sonatype.org:443/, is version 2.14.15-01 and edition "Professional"
[INFO] RC-Releasing staging repository with IDs=[orgeclipseee4j-1001]

Waiting for operation to complete...
..

[INFO] Released
```

Reply:
jsonp 2.0.0-RC1 is in staging so version was updated.